script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -462,27 +462,27 @@ It's as though he were grinning in agreement with Arcueid's provocation.
 [sha:b8aaf187460b84ead817cb5a6b19a3c96c926449]{
 -- Page 548, Offset 10071.
 -- 「最も強きものとは千変にして<偏|かたよ>らぬもの。
-"The strongest ones are those who continue to adapt and do not remain locked in their ways.
+"Those that strong be... retain unfaltering through vicissitudes do.
 }
 [sha:52f1281e1a8e19a264a2c690652cc29cab9f9ace]{
 -- Page 548, Offset 10072.
 -- ……貴様はまさにソレだ。そこまで衰退していながら、まだおれを上回る余力がある。
-...You are exactly that. Even diminished as you are, your remaining power still surpasses my own.
+And that is truly what you are... Though diminished your power may be... above mine it is nonetheless.
```

Comment:
Is Vlov actually speaking this stiltedly here? It doesn't really read that way to me

Reply:
It's hard to tell from the text, but listen to him talk.
He talks very, very slowly, almost at all times.
In MBTL I tend to sprinkle in these ... to give his dialogue that same flavor.